sync.py

```diff
@@ -21,15 +25,31 @@ class SyncS3():
     AWS_ACCESS_KEY = args['key']
     AWS_SECRET_KEY = args['secret']
     BUCKET_NAME = args['bucket']
-    DATA_FOLDER_PATH = 'data/'
+    REGION = args['region']
+    DATA_FOLDER_PATH = args['target'] or 'data/'
 
     def _get_s3_bucket(self):
         '''
-        Create an S3 connection to the department bucket
+        Create an S3 connection to the bucket
         '''
-        conn = S3Connection(self.AWS_ACCESS_KEY, self.AWS_SECRET_KEY)
-        bucket = conn.get_bucket(self.BUCKET_NAME)
-        return bucket
+        if self.REGION is None:
+            conn = S3Connection(self.AWS_ACCESS_KEY, self.AWS_SECRET_KEY)
+        else:
+            # Bucket names with '.' need special treatment
+            if '.' in self.BUCKET_NAME:
+                conn = connect_to_region(
+                    self.REGION,
+                    aws_access_key_id=self.AWS_ACCESS_KEY,
+                    aws_secret_access_key=self.AWS_SECRET_KEY,
+                    calling_format=OrdinaryCallingFormat())
+            else:
+                conn = connect_to_region(
+                    self.REGION,
+                    aws_access_key_id=self.AWS_ACCESS_KEY,
+                    aws_secret_access_key=self.AWS_SECRET_KEY)
+
+        return conn.get_bucket(self.BUCKET_NAME, validate=False)
```

Comment:
@metaodi Yes, this is about boto/boto#2836 Good catch on the validate=False , was a test that I made and not intended to part of the PR.